packages/client/hmi-client/src/workflow/ops/document/tera-document-operation-drilldown.vue

```diff
@@ -0,0 +1,170 @@
+<template>
+	<tera-drilldown :title="node.displayName" @on-close-clicked="emit('close')">
+		<div>
+			<tera-drilldown-section :is-loading="fetchingPDF">
+				<tera-pdf-embed v-if="pdfLink" :pdf-link="pdfLink" :title="document?.name || ''" />
+				<!-- <tera-text-editor v-else-if="view === DocumentView.TXT" :initial-text="docText" /> -->
+			</tera-drilldown-section>
+			<tera-drilldown-preview hide-header>
+				<h5>{{ document?.name }}</h5>
+
+				<h7 class="clamp-text">{{ document?.text }}</h7>
+				<Accordion multiple :active-index="[0, 1, 2]">
+					<AccordionTab v-if="!isEmpty(clonedState.equations)">
+						<template #header>
+							<header>Equation Images</header>
+						</template>
+						<tera-expandable-panel
```

Comment:
What do you think naming it like what you will name it in the `types/workflow.ts`?

```suggestion
						<tera-asset-block
```

Reply:
agreed, renamed